Drivers/common/drivers_config/inc/drivers_config.hpp

```diff
@@ -17,4 +17,6 @@ extern SBUSReceiver* sbus_pointer;
 extern UARTDevice* pRFD900;
 extern CircularBuffer *rfd900_circular_buffer;
 
+#define RFD900_BUF_SIZE 280
```

Comment:
why did we choose the maximum limit of possible mavlink messages instead of making the buffer size larger which allows it to be able to hold more messages at a time? The ring buffer should be meant to temporally hold more messages when the processor is busy doing other stuff.

Reply:
Last when I was talking to Yarema he mentioned that he was likely going to update this to be 10x (or at least some x) the max mavlink message but I didn't touch it since I was just moving the constant from the cpp file to the hpp file. I'm happy to make it larger right now if @Yaremadzulynsky could suggest which size to use

Reply:
Yes. Can you make the buffer size be something like 40*max mavlink message? Thx.